modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js

```diff
@@ -12,15 +12,17 @@
  */
 
 /**
- * Editor UI show method. Only works in inline mode.
+ * Reveals the editor toolbar and menu bar for inline editors. Can be used to show
```

Comment:
This is technically incorrect. It can be used to show the inline UI at any time, not just when using the `toolbar_persist: true` setting.

Reply:
I should add the difference that setting makes, is it'll ensure the UI doesn't appear again when the editor is focused. So it does a different thing and isn't directly related to this new API.

Reply:
Another point I just realised that was incorrect in this edit, the API hides the entire UI, not just the menu bar and toolbar. So it'll hide notifications, dialogs, etc...

Reply:
_Can be_, as in, doesn't need to be, but can be. ;)

Reply:
It's still misleading though, as it sounds like you need to use that setting to use these APIs.

Reply:
Fixed in: https://github.com/tinymce/tinymce/pull/6035/commits/2bd5497b791e32a286e15824ab3f6bcb085fbe85

Reply:
@lnewson - fixed? https://github.com/tinymce/tinymce/pull/6035/commits/5a851849e89413cdf2679cff1a897b0c9f95dea9